openedx/core/djangoapps/credit/api/provider.py

```diff
@@ -108,6 +108,15 @@ def get_credit_provider_info(request, provider_id):  # pylint: disable=unused-ar
     return JsonResponse(credit_provider_data)
 
 
+def clean_newlines(s):  # pylint:disable=invalid-name
+    """
+    Normalize newline characters.
+
+    Use "\r\n" as the newline character.
+    """
+    return s.replace('\r\n', '\n').replace('\r', '\n').replace('\n', '\r\n')
```

Comment:
The fix would be to use `re.replace`, and match any number of chars that are in `[\r\n]`.